requirements/requirements.txt

```diff
@@ -1,4 +1,4 @@
-ingenialink @ git+https://github.com/ingeniamc/ingenialink-python@develop
+ingenialink @ git+https://github.com/ingeniamc/ingenialink-python@bugfix/INGK-687-improve-servo-enable-disable
```

Comment:
Maybe we should wait for this ingenialink branch to be merged.

Reply:
Agreed. The idea was to revert this change before merging.